### PR TITLE
Documentation: Documentation update for Virtualbox appliance generation

### PR DESCRIPTION
**Pull request by Valtis**
Updated documentation to take account the new dsk.ova format.

Depends on https://github.com/ostroproject/meta-ostro/pull/82

Signed-off-by: Erkka Kääriä <erkka.kaaria@intel.com>

[ci skip]

### CODE REVIEW

**Comment by gvancuts** [comment]
It's [skip ci] not [ci skip]... but too late now anyway :-)

File: doc/howtos/booting-and-installation.rst [hunk 1]
```diff
@@ -14,8 +14,13 @@ Two images are of interest for this process (depending if you're using real hard
     and at least one ext4 partition (rootfs).  For details on disk layout
     see the associated :file:`.json` file in the same directory as the image file.
 
+:file: `.dsk.ova`
```
Comment (gvancuts) [inline doc/howtos/booting-and-installation.rst]: I think there needs to be be no space here, i.e. :file:`dsk.ova`

File: doc/howtos/booting-and-installation.rst [hunk 2]
```diff
@@ -14,8 +14,13 @@ Two images are of interest for this process (depending if you're using real hard
...
+    A pre-packaged Virtualbox\* Virtual Machine appliance file that can be directly imported
```
Comment (gvancuts) [inline doc/howtos/booting-and-installation.rst]: s/Virtualbox/VirtualBox

File: doc/howtos/booting-and-installation.rst [hunk 3]
```diff
@@ -14,8 +14,13 @@ Two images are of interest for this process (depending if you're using real hard
     and at least one ext4 partition (rootfs).  For details on disk layout
     see the associated :file:`.json` file in the same directory as the image file.
 
+:file: `.dsk.ova`
+    A pre-packaged Virtualbox\* Virtual Machine appliance file that can be directly imported 
+    to Virtualbox\*
```
Comment (gvancuts) [inline doc/howtos/booting-and-installation.rst]: s/Virtualbox/VirtualBox

File: doc/howtos/booting-and-installation.rst [hunk 4]
```diff
@@ -351,14 +356,23 @@ the partition tables are wiped out.
...
 in the binary release directory (on https://download.ostroproject.org), or as the result of doing your
 own build from source.  As with the other examples above, we recommend you start with the "dev" image.
 
 #. If you have not already done so, download and install VirtualBox (version 5.0.2 or later)
-   on your development system from https://www.virtualbox.org/wiki/Downloads. VirtualBox uses
-   VDI as its native disk image format so you’ll be using that file instead of the .dsk file used
-   with real hardware platforms.
+   on your development system from https://www.virtualbox.org/wiki/Downloads.
+#. Open the VirtualBox program and select "File > Import appliance..."
+#. Click the folder icon in "Import virtual appliance" window, select the .ova file that you
```
Comment (gvancuts) [inline doc/howtos/booting-and-installation.rst]: s/.ova/``.ova``

File: doc/howtos/booting-and-installation.rst [hunk 5]
```diff
@@ -351,14 +356,23 @@ the partition tables are wiped out.
...
+   downloaded or created and select "Open". Click next.
+#. A window opens that lists the details of the appliance that you are about to import. Click import.
+#. Virtualbox will now import the virtual machine. After the import is finished, the Ostro OS virtual
```
Comment (gvancuts) [inline doc/howtos/booting-and-installation.rst]: s/Virtualbox/VirtualBox

File: doc/howtos/building-images.rst [hunk 6]
```diff
@@ -164,13 +164,22 @@ For EFI platforms, you can produce different types of images:
 .dsk:
     The basic format, written to a block device to create a bootable image.
 
+.dsk.ova:
+    Pre-packaged Virtualbox* virtual machine, for running Ostro OS inside
```
Comment (gvancuts) [inline doc/howtos/building-images.rst]: s/Virtualbox/VirtualBox

File: doc/howtos/building-images.rst [hunk 7]
```diff
@@ -164,13 +164,22 @@ For EFI platforms, you can produce different types of images:
...
 
+.dsk.ova:
+    Pre-packaged Virtualbox* virtual machine, for running Ostro OS inside 
+    a Virtual Machine
+
 .dsk.vdi:
-    VirtualBox* format, for running Ostro OS inside a Virtual Machine.
+    VirtualBox* hard drive format, for running Ostro OS inside a Virtual 
+    Machine. Requires manually creating the Virtual Machine through the
+    Virtualbox* user interface. The .dsk.ova format is recommended over
```
Comment (gvancuts) [inline doc/howtos/building-images.rst]: s/.dsk.ova/``.dsk.ova``

File: doc/howtos/building-images.rst [hunk 8]
```diff
@@ -164,13 +164,22 @@ For EFI platforms, you can produce different types of images:
...
+    this.
+
 
 compressed formats:
     Same as above, only compressed, to reduce (final) space occupation
     and speed up the transfer between systems of the Ostro OS image.
-    Notice that the creation of compressed images will require additional
+    Notice that .dsk.ova files are already compressed.
```
Comment (gvancuts) [inline doc/howtos/building-images.rst]: s/.dsk.ova/``.dsk.ova``

File: doc/howtos/booting-and-installation.rst [hunk 9]
```diff
@@ -351,14 +356,23 @@ the partition tables are wiped out.
 Running Ostro OS in a VirtualBox\* VM
 ======================================
 
-You can run an Ostro OS image within a VirtualBox virtual machine by using the pre-built ``.vdi`` file found
+You can run an Ostro OS image within a VirtualBox virtual machine by using the pre-built ``.ova`` file found
```
Comment (barbieri) [inline doc/howtos/booting-and-installation.rst]: @kad can we have "latest" symlinks to all of these so we can point them here? Like latest ova/vdi/dsk.xz

**Comment by Valtis** [comment]
Updated PR.

File: doc/howtos/booting-and-installation.rst (hunk 9 above)
Comment (kad) [inline doc/howtos/booting-and-installation.rst]: we have for latest automated builds. e.g.  https://download.ostroproject.org/builds/ostro-os/latest/images/intel-corei7-64/ostro-image-noswupd-intel-corei7-64.dsk.vdi.zip
but maintaining those names in different places might be not good thing to do.

Comment (barbieri) [inline doc/howtos/booting-and-installation.rst]: I mean for releases, latest would point to the latest milestone released.

**Comment by pohly** [comment]
Looks good to me and I guess @gvancuts and @barbieri also didn't have any objections (the comment from @barbieri seems more about CI deployment than the actual PR), so I'm merging it now that PR #82 is merged.

**Comment by gvancuts** [comment]
@pohly , I did not have any objection @Valtis had addressed all my comments in the updated PR, thanks!

**Pull request merged**